### ISSUE TITLE
after .10 upgrade app does not start any more

### ISSUE THREAD

**Issue by dohomi** [nuxt/nuxt.js#441]
Hello,

I am receiving following error on starting my app:
```
[nuxt.js] Cannot load components TypeError: Cannot read property 'getters' of undefined
    at _callee$ (eval at <anonymous> 
```
I use vuex and set some getters but I rechecked the `getters` existence and but I have no idea where this error comes from. Before the upgrade everything worked as expected. Any ideas?

**Comment by dohomi** [comment]
My main concern is: I have no idea how to debug this error. I can't set any breakpoint to see which components is breaking because its inside of nuxt bundle:

```
[nuxt.js] Cannot load components TypeError: Cannot read property 'getters' of undefined
    at _callee$ (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:367:1), <anonymous>:15:34)
    at tryCatch (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:672:1), <anonymous>:64:40)
    at Generator.invoke [as _invoke] (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:672:1), <anonymous>:299:22)
    at Generator.prototype.(anonymous function) [as next] (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:672:1), <anonymous>:116:21)
    at step (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:367:1), <anonymous>:6:191)
    at eval (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:367:1), <anonymous>:6:437)
    at eval (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:367:1), <anonymous>:6:99)
    at eval (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:367:1), <anonymous>:32:21)
    at promisify (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:346:1), <anonymous>:95:15)
    at promiseSeries (eval at <anonymous> (http://localhost:3333/_nuxt/nuxt.bundle.b0190671b3eaf9cf79ab.js:346:1), <anonymous>:76:10)
```

**Comment by Atinux** [comment]
It should be fixed with the [0.10.2](https://github.com/nuxt/nuxt.js/releases/tag/v0.10.2) release. Sorry about this @dohomi

**Comment by dohomi** [comment]
great all back! thanks for this quick fix and the highly improved released!
